### PR TITLE
80-85 char length line formatting for better docs embedding.

### PR DESCRIPTION
**Pull request by kashimiz**
Per request of @tdykstra on [this docs pull request](https://github.com/MicrosoftDocs/azure-docs-pr/pull/34331#pullrequestreview-104381919), code files should have a character line limit of 80-85 to prevent horizontal scrolling when viewed as embeds in documentation.

### CODE REVIEW

File: samples/csx/E2_BackupSiteContent/run.csx [hunk 1]
```diff
@@ -2,15 +2,18 @@
 
 public static async Task<long> Run(DurableOrchestrationContext backupContext)
 {
-    string rootDirectory = Environment.ExpandEnvironmentVariables(backupContext.GetInput<string>() ?? "");
+    string rootDirectory = Environment.ExpandEnvironmentVariables(
```
Comment (tohling) [inline samples/csx/E2_BackupSiteContent/run.csx]: Undo all the auto-correction that was done to the spacing.

**Comment by cgillum** [comment]
80-85 characters feels too restrictive. I understand the need to make this narrower for documentation, but I worry it could make code in the editor harder to read. Typically I try to do no more than 120 characters. Is it possible to compromise on this a bit? What screen layout are we optimizing for?

**Comment by tdykstra** [comment]
@cgillum The shorter line length is only a nice-to-have for readability, not a requirement. Whether you go with 80-85 or 120 characters is up to you and @kashimiz . Should we close this PR and leave the code as-is? Or merge it after restoring some of the long lines where chopping them compromised code readability in VS?

**Comment by cgillum** [comment]
Let me take another pass through this. There are pros and cons for both sides. I think it makes sense to do it on a case-by-case basis. I just wish GitHub had an option to show line lengths. :)

**Comment by cgillum** [comment]
Actually @kashimiz did you automate this using a tool (like Resharper)? If so, going at this case-by-case might be counter-productive. Would it make sense to just compromise at 100 characters?

One general piece of feedback is that I'd like to avoid using `string.Format` over `$"..."`, especially if we can just drop the string onto a new line to save some characters.

**Comment by kashimiz** [comment]
I didn’t (although I wish I had.)

Re: string.Format, I went there because string interpolation broke the line length limit I’d imposed, even when dropping the string to a new line.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by cgillum** [comment]
I assume the motivation for shorter line lengths is primarily for smaller devices. I went through the documentation on my iPhone in landscape mode and horizontal scrolling wasn't a problem for the majority of the code snippets. I think having column limits is a good idea, but 80-85 is too restrictive IMO. I could live with 120.

For now, let's close this PR as it has gotten stale and we have several new samples which have since been added. I'm open to revisiting this at a later time.

**Pull request closed without merging**